src/components/ApplicationDetails/__data__/WorkflowComponentsData.ts

```diff
@@ -140,6 +140,7 @@ export const mockComponentsData = [
           type: 'Created',
         },
       ],
+      lastPromotedImage: 'quay.io/redhat-appstudio/user-workload:test-ns-test-dotnet60',
       containerImage: 'quay.io/redhat-appstudio/user-workload:test-ns-test-dotnet60',
```

Comment:
should not here be 61?

Reply:
It is expected. The status.lastPromotedImage is not always same with spec.containerImage.